### PR TITLE
add collection doc for classic websites

### PR DESCRIPTION
**Pull request by yingjingyang**

### CODE REVIEW

**Review by jhfnetboy — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -64,6 +64,9 @@
 - [MIT Online Course: Blockchain and Money by Prof. Gary Gensler](https://ocw.mit.edu/courses/sloan-school-of-management/15-s12-blockchain-and-money-fall-2018/video-lectures/)
 
 
+**经典网站收藏**  
+- 收录经典的网站: ClassicWebSites.md
```
Comment (xrdavies) [inline README.md:68]: Please add hyperlink instead of filename.

**Review by xrdavies — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yingjingyang) [reply README.md]: changed

Comment (xrdavies) [reply README.md]: Thank you!

**Review by xrdavies — APPROVED** [review]
Looks good to me

**Review by liberhe — APPROVED** [review]

**Pull request merged**